### ISSUE TITLE
Sourcemap warning: Unexpected generated column decrease

### ISSUE THREAD

**Issue by TheSpyder** [evanw/esbuild#378]
I'm working with [SlateJS](https://github.com/ianstormtaylor/slate) in an app and am evaluating a switch to esbuild for bundling. However when I turn on `--sourcemap` in esbuild, I get a warning about Slate's sourcemap:

```
node_modules/slate/dist/index.es.js.map:1:162026: warning: Bad "mappings" data in source map at character 313: Unexpected generated column decrease
..."isLast","isInlineOrText","newChild","previousPath","oldAncestors","newAncestors","nextPath"],"mappings":";;;;AAAe,SAAS,eAAe,CAAC,GAAG,EAAE,GAAG,EAAE,KAAK,EAAE;EACvD,IAAI,GAAG,IAAI,GAAG,EAAE;IACd,MA...
                                                                                                 ~~~~~~~~~~
1 warning
```

[Slate uses rollup](https://github.com/ianstormtaylor/slate/blob/master/config/rollup/rollup.config.js#L170-L172) to generate the `index.es.js` sourcemap, so I am assuming this is an esbuild bug, but if not I'm happy to log this with the appropriate project.

Replication repo: https://github.com/TheSpyder/esbuild-slate-sourcemap

**Comment by evanw** [comment]
Thanks for the report! This should be fixed in version 0.6.34.

**Comment by TheSpyder** [comment]
Yes, the update fixes the problem for me. Thank you for the fast turnaround!
